extra_data/reader.py

```diff
@@ -255,6 +263,14 @@ def _get_source_data(self, source):
         if source not in self._sources_data:
             raise SourceNameError(source)
 
+        sd = self._sources_data[source]
+
+        if sd.is_legacy:
+            warn(f"{source} is a legacy name for {self.legacy_sources[source]}. "
+                 f"Access via this name will be removed at a future data.",
```

Comment:
nitpicking comment
```suggestion
                 "Access via this name will be removed at a future data.",
```

Also, I assume here you don't mean at a future date but for future data.?

Reply:
I did originally mean *date*, but actually referencing future *data* is a good idea. We're not going to remove the legacy name from existing files, just stop adding it. Thanks!